src/App.vue

```diff
@@ -1,21 +1,29 @@
 <script setup lang="ts">
+import { computed } from 'vue'
+import Login from './components/Login.vue'
 import Navbar from './components/Navbar.vue'
-import Main from './components/Main.vue'
+import { useSessionToken } from './composables/useSessionToken'
+import { useUserDetails } from './composables/useUserDetails'
+
+const token = useSessionToken()
+const user = useUserDetails()
+const loggedIn = computed(
+	() => token.value && user.value?.role === 'Administrator'
+)
```

Comment:
Since `useUserDetails` already calls `useSessionToken` internally; do we really need the call the latter again over here?

Reply:
We actually don't. Good call.